main.js

```diff
@@ -33,12 +33,24 @@ for(let i = 0; i < testMas2.length; i++){
         console.log(testMas2[i]);
     }
 }
+console.log('СПОСОБ №2:')
+for(let i of testMas2){
+    if(testMas2[i] > 5 && testMas2[i] < 10){
+        console.log(testMas2[i]);
+    }
+}
```

Comment:
read about for of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of